conf/nutch-default.xml

```diff
@@ -1896,4 +1896,33 @@ CAUTION: Set the parser.timeout to -1 or a bigger value than 30, when using this
   </description>
 </property>
 
+<!-- index-links plugin -->
+
+<property>
+  <name>outlinks.host.ignore</name>
+  <value>false</value>
+  <description>
+    Ignore the outlinks that point out to the same host as the URL being indexed. 
+    By default all outlinks are indexed.
+  </description>
+</property>
+
+<property>
+  <name>inlinks.host.ignore</name>
+  <value>false</value>
+  <description>
+    Ignore the inlinks coming from the same host as the URL being indexed. By default 
+    all inlinks are indexed.
+  </description>
```

Comment:
If db.ignore.internal.links == true (that's the default) this option wouldn't change anything because host-internal-inlinks aren't even stored in linkdb. Could be worth to place a reference to db.ignore.internal.links.